src/sentry/static/sentry/app/components/emptyStateWarning.tsx

```diff
@@ -24,7 +23,7 @@ const EmptyStateWarning = ({
   small ? (
     <EmptyMessage className={className}>
       <SmallMessage>
-        {withIcon && <StyledIconWarning color={theme.gray500} size="lg" />}
+        {withIcon && <StyledIconWarning color="gray500" size="lg" />}
```

Comment:
I think one benefit of this change is that we don't need to import theme which yields some webpack boilerplate savings.